addons/website_profile/views/website_profile.xml

```diff
@@ -0,0 +1,361 @@
+                                        <button type="button" class="close" data-dismiss="alert">x</button>
+                                        <p>Please enter a valid email address in order to receive notifications from answers or comments.</p>
+                                    </div>
+                                    <label class="mb-1 text-primary" for="user_email"><b>Email</b></label>
+                                    <div>
+                                        <input type="text" class="form-control" name="email" id="user_email" required="True" t-attf-value="#{user.partner_id.email}"/>
+                                    </div>
+                                </div>
+                                <div class="form-group col-6">
+                                    <label class="mb-1 text-primary" for="user_city"><b>City</b></label>
+                                    <div>
+                                        <input type="text" class="form-control" name="city" id="user_city" t-attf-value="#{user.partner_id.city or ''}"/>
+                                    </div>
+                                </div>
+                                <div class="form-group col-6">
+                                    <label class="mb-1 text-primary"><b>Country</b></label>
+                                    <div>
+                                        <select class="form-control" name="country">
+                                            <option value="">Country...</option>
+                                            <t t-foreach="countries or []" t-as="country">
+                                                <option t-att-value="country.id" t-att-selected="country.id == user.partner_id.country_id.id"><t t-esc="country.name"/></option>
+                                            </t>
+                                         </select>
+                                    </div>
+                                </div>
+                                <div class="form-group col-12">
+                                    <label class="mb-1 text-primary" for="description"><b>Biography</b></label>
+                                    <textarea name="description" id="description" style="min-height: 120px" required="True"
+                                        class="form-control load_editor"><t t-esc="user.partner_id.website_description"/></textarea>
+                                </div>
+                                <div class="col">
+                                    <button type="submit" class="btn btn-primary btn">Update</button>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </form>
+        </div>
+    </template>
+
+    <!-- Profile Page -->
+    <template id="user_detail_main">
```

Comment:
Naming: just for coherence, I would keep the same naming as methods / routes / templates. For example
user_profile_main, user_profile_edit_main, ... that way it is easier to find. You already have main/header/content within same namespace which is great. Let us be (pixel)perfect !